pyproject.toml

```diff
@@ -22,7 +22,7 @@ exclude_lines = [
 
 [tool.poetry]
 name = "cryptojwt"
-version = "1.7.1"
+version = "1.7.2"
```

Comment:
This is an API change/update, should be bumped to 1.8.0

Reply:
OK